addons/event_crm/models/event_question_answer.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models
+
+
+class EventQuestionAnswer(models.Model):
+    _inherit = 'event.question.answer'
+
+    def action_add_rule_button(self):
+        self.ensure_one()
+        action = self.env['ir.actions.actions']._for_xml_id('event_crm.event_lead_rule_answer_action')
+        action['context'] = {
+            'default_name': self.name,
+            'default_lead_user_id': self.env.user.id,
+            'default_event_registration_filter': [
+                '&',
+                ('registration_answer_ids.question_id', 'ilike', self.question_id.title),
```

Comment:
As it is a translated field, this is going to fail :) can't we use question id ? If we have 3 "name" well tant pis.